logging/src/main/java/tech/pegasys/teku/logging/EventLogger.java

```diff
@@ -56,6 +56,14 @@ public void epochEvent(
     info(epochEventLog, Color.GREEN);
   }
 
+  public void nodeSlotDriftEvent(final UnsignedLong oldSlot, final UnsignedLong newSlot) {
+    final String driftEventLog =
+        String.format(
+            "Drift Event *** Current slot: %s, previous slot: %s",
```

Comment:
Drift Event probably isn't the right wording here.  Probably just "Skipped Slots"?  I don't think we always need Event in there and possibly should just remove them all as redundant.

Reply:
skipped slots is too long... unless i move all the messages ?

Reply:
I replaced with 'Miss Slots' - thoughts?